OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -352,16 +352,10 @@ public void SetSoundLooping(bool looping, ISound sound)
 
 		~OpenAlSoundEngine()
 		{
-			Dispose(false);
+			Dispose();
 		}
 
 		public void Dispose()
-		{
-			Dispose(true);
-			GC.SuppressFinalize(this);
-		}
-
-		void Dispose(bool disposing)
```

Comment:
Revert these changes. The correct fix here is that the `disposing` should guard managed cleanup. i.e. we should use it to guard the StopAllSounds call.
```c#
if (disposing)
    StopAllSounds();
```